src/DropdownInput.js

```diff
@@ -6,9 +6,8 @@
 
 'use strict';
 
-var React = require('react/addons');
+var React = require('react');
 var ReactBootstrap = require('react-bootstrap');
-var joinClasses = require('react/lib/joinClasses');
 var cx = require('classnames');
 
 var BootstrapMixin = ReactBootstrap.BootstrapMixin;
```

Comment:
@ravloony - Hey Tom: I'm glad you attempted the fix - however, I'm trying to use this lib with React 14 and React-bootstrap 0.28, and I believe all this mixin stuff has gotten dropped. Just something to be aware of, going forward. This seems to be common issues with all of these search box components, that folks are not upgrading them very quickly.